lib/vault/test/index.test.ts

```diff
@@ -1,11 +1,10 @@
 import { describe, it, beforeAll, beforeEach, afterAll, jest, expect } from '@jest/globals'
 import { VaultEnvVars } from '../src/index'
 import fetch from '@financial-times/n-fetch'
-import { mocked } from 'ts-jest/utils'
 import fs from 'fs'
 import winston, { Logger } from 'winston'
 
-const logger = (winston as unknown) as Logger
+const logger = winston as unknown as Logger
```

Comment:
**thought**: i feel like Prettier keeps flipflopping this syntax, i'm sure i've seen the reverse of this diff recently, am i mad?

Reply:
This is the change the [Prettier PR](https://github.com/Financial-Times/dotcom-tool-kit/pull/614/files#diff-0ab615cc73003f4494a8b02ba1c7b5fd627469782fb930afce15a35a38396131) makes too so I think it's okay here. I can't find any PRs going the other way anyway 🤷